gamemodes/terrortown/gamemode/server/sv_player_ext.lua

```diff
@@ -195,19 +195,21 @@ end
 ---
 -- Gives a specific @{ITEM} (if possible)
 -- @param string cls
+-- @return @{ITEM} or false
 -- @realm server
 -- @internal
 function plymeta:AddEquipmentItem(cls)
 	local item = items.GetStored(cls)
 
-	if not item or item.limited and self:HasEquipmentItem(cls) then return end
+	if not item or item.limited and self:HasEquipmentItem(cls) then return false end
```

Comment:
why return false here? if `nil` is returned, a simple `IsValid` check is enough

Reply:
Sure. I am personally not a fan of returning nil (I think it's bad code style), but Ill change it. IsValid(false) should return false, so it makes no difference whatsoever, but return nil is probably more consistent with other functions.

Reply:
I see where you're coming from. But in lua or other untyped languages I think it is not a bad style. But maybe someone else has a different opinion